### PR TITLE
adding new test scenarios

### PR DESCRIPTION
**Pull request by benjisclowder**
## **Description**

_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:_
_1. What is the reason for the change?_
_2. What is the improvement/solution?_

Adding test scenarios for manual QA of the tokens flows.

## **Manual testing steps**

_1. Step1:_
_2. Step2:_
_3. ..._

## **Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change._

### **Before**

_[screenshot]_

### **After**

_[screenshot]_

## **Related issues**

_Fixes #???_

https://github.com/MetaMask/metamask-extension/issues/20256 and https://github.com/MetaMask/metamask-extension/issues/20255

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained:
  - [ ] What problem this PR is solving.
  - [ ] How this problem was solved.
  - [ ] How reviewers can test my changes.
- [ ] I’ve indicated what issue this PR is linked to: Fixes #???
- [ ] I’ve included tests if applicable.
- [ ] I’ve documented any added code.
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)).
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by chloeYue — DISMISSED** [review]
LGTM! Thanks!

File: test/scenarios/4. tokens/import erc721 token origin dapp.csv [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+4,Open the test dapp in another tab.,https://metamask.github.io/test-dapp/,,
+5,Proceed to connect with MetaMask. ,,,
+6,Connect with the current account. ,,,
+7,Deploy the NFT contract. ,,,
+8,Confirm the transaction and wait for it to complete.,,,
+9,Proceed to mint NFT. ,,,
+10,Confirm the transaction and wait for it to complete.,,,
+11,Switch to the wallet and open the activity list item.,,"The transaction status, recipient's address, nonce, amount, gas and total are shown in the item details.",
+12,Proceed to view the transaction on the block explorer.,,The block explorer opens in a new tab.,
+13,"In the test dapp, proceed to Watch NFT. ",,,
```
Comment (PeterYinusa) [inline test/scenarios/4. tokens/import erc721 token origin dapp.csv:14]: Can you add details here to ensure the watch NFT input field is filled in with the ID of the NFT to watch?

File: test/scenarios/9. metrics/send event metrics.csv [hunk 2]
```diff
@@ -13,6 +13,7 @@ Chrome: chrome://extensions/",The extensions page is displayed.,
...
 9,Inspect background.html using the browsers developer tools.,,The browsers developer tools window opens.,
 10,Select the Network tab.,,,
-11,Close the browsers extension page and switch back to the wallet.,,,
-12,Proceed to General in the Settings menu.,,"A request to Sentry is made - https://api.segment.io/v1/batch.
-The request payload is valid JSON and contains the name General Settings Page.",
\ No newline at end of file
+11,Keep the background console open and head over to the wallet.,,,
+12,Proceed to General in the Settings menu.,,"A request to Sentry is made in the networks tab under payload - https://api.segment.io/v1/batch.
+The request payload is valid JSON and contains the name General Settings Page.",
+,,,,
```
Comment (PeterYinusa) [inline test/scenarios/9. metrics/send event metrics.csv:19]: Can you remove this empty row?

**Review by PeterYinusa — CHANGES_REQUESTED** [review]

File: test/scenarios/9. metrics/send event metrics.csv (hunk 2 above)
Comment (benjisclowder) [reply test/scenarios/9. metrics/send event metrics.csv]: done!

File: test/scenarios/4. tokens/import erc721 token origin dapp.csv (hunk 1 above)
Comment (benjisclowder) [reply test/scenarios/4. tokens/import erc721 token origin dapp.csv]: done!

**Review by PeterYinusa — APPROVED** [review]

**Review by chloeYue — APPROVED** [review]

**Pull request merged**